config/config.go

```diff
@@ -886,6 +888,10 @@ type AzureSDConfig struct {
 	XXX map[string]interface{} `yaml:",inline"`
 }
 
+type MDNSConfig struct {
+	Enabled bool `yaml:"enabled"`
```

Comment:
I'm not a fan of boolean config options and I don't think we should provide enabled switches. Can we just leave the config empty?